templates/site/js/node.js

```diff
@@ -10,6 +10,19 @@ const converters = {
         node_description_title.textContent = val;
         return node_description_title;
     },
+    "registration_status": (val) =>{
+        let registration_status_title = document.createElement("span");
+        registration_status_title.innerText = "Registration Status: ";
+        registration_status_title.style.fontWeight = "bold";
```

Comment:
Styling should be done with css whenever possible

Reply:
Moved bold styling to css